spec/requests/grantees_spec.rb

```diff
@@ -0,0 +1,130 @@
+require "rails_helper"
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
```

Comment:
[suggestion] please delete all commented code, and code related to a route that is not the show route